src/test/java/org/owasp/esapi/logging/java/JavaLogFactoryTest.java

```diff
@@ -48,83 +43,6 @@ public class JavaLogFactoryTest {
     @Rule
     public ExpectedException exEx = ExpectedException.none();
 
-    @Test
```

Comment:
So, we're not going to try to test the case if the `ConfigurationException` is thrown if the file **esapi-java-logging.properties** is found. As long as you tested all the cases separately, I guess I'm okay with that, otherwise the testing gets messy because you have to create the file, test, delete it, and test again.

Reply:
Yeah, I went light on the testing with this change as we'd have to be testing a negative condition.

I am relying on the implied tested state from all of the other tests - knowing that the properties file has been removed and the configuration has changed, all existing tests continue to function.

Reply:
Sorry I'm late to the party.  Kids birthday this weekend and family's in town.  

Regression tests FTW.  I have no issues here.